### ISSUE TITLE
[Feature] Full support of bibliography and citations

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/onlyoffice.github.io#112]

**Comment by tvrbanec** [comment]
> @parrenin
> 
> > OnlyOffice does not support the Zotero bibliography manager.
> 
> It does, please [follow the link](https://github.com/ONLYOFFICE/sdkjs-plugins/tree/feature/zotero/zotero) to find an example of how such plugin should work.

Error 404 (page not found)

**Comment by giovanna-deluca** [comment]
Trying to put MS Word aside for my academic activity, I installed OnlyOffice and the Zotero plugin.
I just landed here while looking for a way to add in-text citations.
Are there still no solutions to this lacking feature? This is forcing me to abandon the idea of using OnlyOffice.

**Comment by tvrbanec** [comment]
Olyoffice manufacturers are not serious at all. They do not understand that what professors do and what and how we teach students, directly influence what students will use later in life and work. As if they did not want to spread their user base.

**Comment by Rosmaninho** [comment]
Just came back to try the plugin and still no way to add citations. It is a shame. 
Such a shame. Even GDocs has this.
OnlyOffice beats LibO in every other regard except this. C'mon devs...

**Comment by meihollander** [comment]
Just wanted to know if any more progress has been made on this front. I'm especially interested in the plugin for the 'Desktop Editors' version.

**Comment by shu0406** [comment]
I got the information by onlyoffice that there soon will be a new beta version of the plugin. "methods needed for the plugin were only added in version 7.3. which was released recently". So I hope that version will support full functionality!

**Comment by meihollander** [comment]
> I got the information by onlyoffice that there soon will be a new beta version of the plugin. "methods needed for the plugin were only added in version 7.3. which was released recently". So I hope that version will support full functionality!

Where can I find the beta version?

**Comment by meihollander** [comment]
> finally...
> 
> https://www.onlyoffice.com/blog/2023/03/meet-updated-zotero-plugin/

Does this new update include in-text citations?

**Comment by jdpipe** [comment]
This plugin has improved greatly now. You can add citations and insert a bibliography. Two issues that I noticed:

1. You can't choose 'in-text' citations (equivalent to \citep and \citet in Latex), so usage is somewhat limited still. Also, numerical citations don't come with any formatting (eg [1], superscript 1, superscript [1], etc).
2. You can't select which Zotero library you want to connect to. It connects to 'My Library' by default, and I can't see any way to connect my document to a different library.

It's a bit improvement, and could start to be used now, albeit with some significant limitations.

Also, I note that the process of installing it is a bit annoying. You have to zip up your own plugin, and it is *not* in fact included with Onlyoffice 7.3.3 as mentioned somewhere.

**Comment by VladimirProkofiev52** [comment]
Hello! @jdpipe, @parrenin.,
When you say "You can't select 'in text'", do you want citations/bibliographies to have the same formatting as your text?
By formatting, I mean font style, size, and etc.

**Comment by sith-on-mars** [comment]
The current version requires a Zotero account. I wish in the future the plugin can support offline use.

**Comment by tvrbanec** [comment]
> The current version requires a Zotero account. I wish in the future the plugin can support offline use.

That is not real and is not working that way in either OS or Office. But the plugin is just not working fully and is not usable at all.

**Comment by leighss** [comment]
Yes, we need the same functionality as LibreOffice and Softmaker TextMaker
Everything else is there (including equation editor which Softmaker doesnt have in Linux)
I agree completely with @tvrbanec - you wouldnt believe the amount of students I have recommended Zotero too (although the uni gives them software provided free (free while they are studying ....) from The Pusher M$

**Comment by lnborise** [comment]
It's great to have the Zotero plug-in, but the is still no solution for in-text citations  :(

> Hello! @jdpipe, @parrenin., When you say "You can't select 'in text'", do you want citations/bibliographies to have the same formatting as your text? By formatting, I mean font style, size, and etc.

No, in-text citation is a format of a citation when only the year of the reference is included in the parentheses:

As was shown already by Smith (1958), <...>

Currently, Zotero in OnlyOffice can only do the following, which is a wrong format for an in-text citation:

As was shown already by (Smith 1958), <...>

We really need a solution for this -- otherwise, it's impossible to use Zotero in OnlyOffice :(

**Comment by VladimirProkofiev52** [comment]
Hello! @parrenin 
Recently, we released plugin version 1.0.7. Please update both the plugin and the editor to the latest versions, as the functionality you requested should now be available there.

**Comment by VladimirProkofiev52** [comment]
Since the Zotero plugin has already been implemented, has several released versions, and includes a wide range of settings, I will close this issue as fixed.

If you have any further suggestions for improving the plugin or encounter any issues, please create a new issue.
